tests/ruby_class/class_2.rb

```diff
@@ -2,4 +2,4 @@ class Processor < AST::Processor
   def initialize(a)
     puts a
   end
-end
\ No newline at end of file
+end
```

Comment:
Not really related to this PR, I think I opened this and my editor instantly fixed it. I can revert if desired :+1: